Automattic/woocommerce-payments
# Allow merchant to set store logo on Platform Checkout settings

Fixes #4257

#### Changes proposed in this Pull Request

* Add a new file field on Platform Checkout settings that updates merchant store logo.









#### Testing instructions





* Access WCPay settings.
* On Platform Checkout, click on `Customize`.
* On Store logo section, click `Upload custom logo`.
* After upload finish, click `Save changes`.
* Reload the page, it should be persisted.

-------------------


**Post merge**
